content/docs/gke/troubleshooting-gke.md

```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
 For more help, try the 
 [general Kubeflow troubleshooting guide](/docs/other-guides/troubleshooting).
 
-## Troubleshooting Cloud Identity-Aware Proxy (Cloud IAP)
+## Troubleshooting kubeflow deployment on GCP
 
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
+* Make sure you are using HTTPS.
+* Check project [quota page](https://console.cloud.google.com/iam-admin/quotas) to see if any service's current usage reached quota limit, increase them as needed.
+* Check [deployment manager page](https://console.cloud.google.com/deployments) and see if there’s a failed deployment.
+* Check if endpoint is up: do [DNS lookup](https://mxtoolbox.com/DNSLookup.aspx) against your IAP url and see if can resolve to correct ip.
```

Comment:
On first usage of "IAP", provide full term:

"Cloud Identity-Aware Proxy (Cloud IAP)"